SPECS/shim/shim.spec

```
@@ -2,15 +2,20 @@
 %define release_number %(echo "%{release}" | cut -d. -f1)
 Summary:        First stage UEFI bootloader
 Name:           shim
-Version:        15.4
-Release:        2%{?dist}
+Version:        15.8
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/rhboot/shim
-# This signed-shim tarball contains the shim binary signed with
-# the Microsoft UEFI CA key
-Source0:        signed-%{name}-%{version}-%{release_number}.tar.gz
+# The below source URL will point to the backing shim source code version used
+# to build the shim binary which has been signed with the MS UEFI CA. This is
+# needed for component governance.
+# The Source0 that gets used is actually the signed shim binary, whose filename
+# is annotated after the '#'. The signed shim binary is named with the following
+# schema to avoid name collisions:
+#   signed-shim<arch>-<version>-<release>.<dist tag>.efi
+Source0:        https://github.com/rhboot/shim/releases/download/15.8/shim-15.8.tar.bz2#signed-shimx64-%{version}-%{release_number}.cm2.efi
```

Might be worth to use RPM's `%{_arch}` macro, so the spec doesn't need any updates, if we happen to start producing shims for other architectures:
```suggestion
Source0:        https://github.com/rhboot/shim/releases/download/15.8/shim-15.8.tar.bz2#signed-shim%{_arch}-%{version}-%{release_number}.cm2.efi
```

This will require re-naming the current file a bit to use "x86_64" instead of "x64".

---

Shims of different architectures are canonically named `shimx64.efi` and `shimaa64.efi`. I was keeping this naming pattern.

---

After further thought, I do like using the rpm %{_arch} macro. Added